src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseFVM.cpp

```diff
@@ -588,11 +588,10 @@ real64 CompositionalMultiphaseFVM::scalingForSystemSolution( DomainPartition & d
                                         globalDeltaPresMax.value,
                                         globalDeltaPresMax.location ) );
   GEOS_LOG_LEVEL_INFO_RANK_0( logInfo::Solution,
```

Comment:
can we do this instead of nested GEOS_FMT:
```
  GEOS_LOG_LEVEL_INFO_RANK_0( logInfo::Solution,
                              GEOS_FMT( "        {}: Max component density change = {:.3f} {} (before scaling) at cell {}",
                                        getName(),
                                        globalDeltaCompDensMax.value,
                                        massUnit,
                                        globalDeltaCompDensMax.location ) );
```

Reply:
I pushed what you suggested... I need a primer in GEOS_FMT.